### PR TITLE
HomeWork 01

### PR DESCRIPTION
**Pull request by andreikasmykou**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "babel": {
+    "presets": [
+      "env"
+    ]
+  },
+  "scripts": {
+    "build": "babel src -d dist",
+    "babel-node": "babel-node --presets=env",
+    "start": "nodemon --exec npm run babel-node -- src/app.js",
+    "lint": "./node_modules/.bin/eslint ./src",
```
Comment (slowpokex) [inline package.json]: Just "lint": "eslint ./src" , because you added eslint to devDeps

File: package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "type": "git",
+    "url": "git+https://github.com/Raerka/nodejs_mentoring.git"
+  },
+  "author": "Andrei Kasmykou <andrei_kasmykou@epam.com>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Raerka/nodejs_mentoring/issues"
+  },
+  "homepage": "https://github.com/Raerka/nodejs_mentoring#readme",
+  "babel": {
```
Comment (slowpokex) [inline package.json]: Maybe need to extract into .babelrc?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "presets": [
+      "env"
```
Comment (slowpokex) [inline package.json]: need to add "stage-2"

File: package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    ]
+  },
+  "scripts": {
+    "build": "babel src -d dist",
+    "babel-node": "babel-node --presets=env",
```
Comment (slowpokex) [inline package.json]: If you added presets to babel config, the preset flag should be omited

File: package.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    "start": "nodemon --exec npm run babel-node -- src/app.js",
```
Comment (slowpokex) [inline package.json]:
```
"start": "node dist/app.js",
"watch": "nodemon src/index.js --exec babel-node"
```

Did you feel differenses?

File: .eslintrc.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    "env": {
+        "es6": true,
+        "browser": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "sourceType": "module"
+    },
+    "rules": {
```
Comment (slowpokex) [inline .eslintrc.js:11]: Please add this rule https://eslint.org/docs/rules/eol-last

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# Program Folders
+.idea/
+conditions/
```
Comment (slowpokex) [inline .gitignore:3]: please add dist folder ignore

**Review by slowpokex — CHANGES_REQUESTED** [review]
Almost good, but for 5 stars need to make some changes

File: package.json (hunk 1 above)
Comment (andreikasmykou) [reply package.json]: Fixed, it works. On skype meeting i will need some explanation about this case.

File: package.json (hunk 2 above)
Comment (andreikasmykou) [reply package.json]: Fixed. Is it best practice to extract everything that you can from package.json, e.g. babel settings or jest settings?

File: package.json (hunk 3 above)
Comment (andreikasmykou) [reply package.json]: Fixed

File: package.json (hunk 4 above)
Comment (andreikasmykou) [reply package.json]: Fixed.

File: package.json (hunk 5 above)
Comment (andreikasmykou) [reply package.json]: Yes.
In first line we run already transpile app.
And in the second we run our app with nodemon which watches any changes in our files and rerun our app.
According to the conditions we needed only two scripts - start and test)))

File: .gitignore (hunk 7 above)
Comment (andreikasmykou) [reply .gitignore:3]: Fixed. dist folder was added.

File: .eslintrc.js (hunk 6 above)
Comment (andreikasmykou) [reply .eslintrc.js:11]: Rule was added. Fixed.

**Review by slowpokex — APPROVED** [review]

**Pull request merged**